docs/src/man/hosting.md

```diff
@@ -214,6 +216,13 @@ When documentation is generated it is stored in one of the following folders:
   are stored in directories named after their tags. These tagged directories are persistent
   and must be manually removed from the `gh-pages` branch if necessary.
 
+The `stable` and `latest` pages are found at:
```

Comment:
Could add one more clarification perhaps: "Unless a custom domain is being used, ...".